src/InvokePlaster.ps1

```diff
@@ -1,4 +1,4 @@
-<#
+﻿<#
```

Comment:
What did you change the encoding to?  UTF-8 with BOM?  Weird that PSSA would think the file was Unicode.